frontend/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,161 @@
+import { Box, Button, Container, Grid, IconButton, makeStyles, Typography } from "@material-ui/core";
+import { Facebook, GitHub } from "@material-ui/icons";
+import Link from "next/link";
+import React from "react";
+import rubberdokLogo from "@public/img/rubberdok_logo_white.svg";
+import Image from "next/image";
+import HallOfFame from "@components/Footer/HallOfFame";
+
+const useStyles = makeStyles((theme) => ({
+  footer: {
+    color: "#b0aca5",
+    background: "#022a2a",
+    paddingBottom: theme.spacing(6),
+    paddingTop: theme.spacing(6),
+    position: "relative",
+    zIndex: 0,
+  },
+  credits: {
+    background: "#021c1c",
+    position: "relative",
+    display: "flex",
+    height: 112,
+    paddingTop: theme.spacing(5),
+    paddingBottom: theme.spacing(5),
+    color: "#b0aca5",
+    [theme.breakpoints.down("sm")]: {
+      height: "unset",
+      flexDirection: "column",
+    },
+  },
+  creditsContent: {
+    [theme.breakpoints.down("sm")]: {
+      flexDirection: "column",
+      alignItems: "center",
+    },
+
+    "& > *": {
+      marginBottom: theme.spacing(2),
+    },
+  },
+  title: {
+    color: "#fff",
+
+    "&:hover": {
+      cursor: "pointer",
+    },
+  },
+  nth: {
+    background: "url('/nth.svg')",
+    backgroundSize: 500,
+    backgroundPosition: "right center",
+    backgroundRepeat: "no-repeat",
+    opacity: 0.25,
+
+    [theme.breakpoints.down("md")]: {
+      width: "100%",
+      left: 0,
+    },
+  },
+  rdLogo: {
+    height: "100%",
+    float: "left",
+
+    [theme.breakpoints.up("md")]: {
+      marginRight: theme.spacing(4),
+    },
+
+    "&:hover": {
+      cursor: "pointer",
+    },
+  },
+  infoBox: {
+    marginTop: -80,
+    marginBottom: 72,
+  },
+}));
+
+const Footer: React.FC = () => {
+  const classes = useStyles();
+
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <>
+      <div>
+        <Box className={classes.footer}>
```

Comment:
A lot of boxes here, could this be refactored? Could add it to the backlog.

Reply:
Add it to the backlog, I think this is something that should rather be part of a bigger styling refactor.